WordPress/Classes/Models/SelfHostedLoginDetails.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+import CryptoKit
+
+// TODO: The majority of this should probably be cross-platform code
+struct SelfHostedLoginDetails {
+    let url: URL
+    let username: String
+    let password: String
+    let XMLRPCEndpoint: URL?
```

Comment:
- (nit) should start with a lowercase letter: `xmlrpcEndpoint`

Reply:
Addressed in d107ed6c42e1ac5b61556d568b7ee8adf0394734